components/ButtonBar/README.md

```diff
@@ -179,79 +179,11 @@ buttonBar.items = items;
 
 ### Color Theming
 
-You can theme a button bar with your app's color scheme using the ColorThemer extension.
-
-You must first add the Color Themer extension to your project:
-
-```bash
-pod 'MaterialComponents/ButtonBar+ColorThemer'
-```
-
-<!--<div class="material-code-render" markdown="1">-->
-#### Swift
-```swift
-// Step 1: Import the ColorThemer extension
-import MaterialComponents.MaterialButtonBar_ColorThemer
-
-// Step 2: Create or get a color scheme
-let colorScheme = MDCSemanticColorScheme()
-
-// Step 3: Apply the color scheme to your component
-MDCButtonBarColorThemer.applySemanticColorScheme(colorScheme, to:buttonBar)
-```
-
-#### Objective-C
-
-```objc
-// Step 1: Import the ColorThemer extension
-#import "MaterialButtonBar+ColorThemer.h"
-
-// Step 2: Create or get a color scheme
-id<MDCColorScheming> colorScheme = [[MDCSemanticColorScheme alloc] initWithDefaults:MDCColorSchemeDefaultsMaterial201804];
-
-// Step 3: Apply the color scheme to your component
-[MDCButtonBarColorThemer applySemanticColorScheme:colorScheme
-     toButtonBar:buttonBar];
-```
-<!--</div>-->
+MDCButtonBar is intended to be themed through the AppBar component's theming extension.
```

Comment:
Tip: A link to https://github.com/material-components/material-components-ios/tree/develop/components/AppBar#theming would be helpful here and throughout.

Reply:
I consolidated the color and typography docs and included a link to app bar theming 👍